gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/services/CreateGraphServiceTest.java

```diff
@@ -53,6 +53,7 @@
 
     @Test
     public void createAccumuloGraph_shouldCallCrdClientWithCreateGraphRequestAndCorrectGraphConfigAndAccumuloEnabled() throws GaaSRestApiException {
+        when(meterRegistry.counter(anyString(), ArgumentMatchers.<String>any())).thenReturn(mockCounter);
```

Comment:
Is this change meant to be in this PR? Just checking?

Reply:
Hi @t92549,   No it is not meant  to be in this PR. The change happend because merge conflict with upstream master.